code/modules/overmap/ships/machines/gas_thruster.dm

```diff
@@ -27,6 +27,16 @@
 		if(use_power)
 			add_overlay(emissive_overlay(icon, "nozzle_idle"))
 
+/obj/machinery/atmospherics/unary/engine/attackby(obj/item/I, mob/user)
+	. = ..()
```

Comment:
Probably want to `return TRUE` to the multitool logic and then call this after that, otherwise you're going to get multiple messages back.

Reply:
Not sure what you're referring to as the "multitool logic" here.

Reply:
The line immediately under this comment.

Reply:
To elaborate: You should handle logic specific to this type before calling `..()`, and you should return TRUE before `..()` if you have already handled an interaction. Otherwise you end up with multiple interactions being completed.